customer-commons/src/main/java/no/unit/nva/customer/model/CustomerMapper.java

```diff
@@ -0,0 +1,82 @@
+package no.unit.nva.customer.model;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import nva.commons.utils.JsonUtils;
+
+public class CustomerMapper {
```

Comment:
This methods of this class should be put to the different objects they generate. See comments below.